apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -6754,6 +6756,44 @@ fate_vm_version_switching(Cfg) ->
     ok.
 
 
+contract_init_on_chain_fate(Cfg) ->
+    state(aect_test_utils:new_state()),
+    ForkHeights   = ?config(fork_heights, Cfg),
```

Comment:
The amount of spaces after the variable jumps from one line to the other. Please fix the formatting